src/core_modules/capture-core/components/DataStore/DataStore.epics.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { ofType } from 'redux-observable';
+import { mergeMap, catchError } from 'rxjs/operators';
+import { EMPTY } from 'rxjs';
+import { fetchDataStore } from './DataStore.actions';
+import { getApi } from '../../d2';
+import { type UseNewDashboard } from './DataStore.types';
+import { appStartActionTypes } from '../../../../components/AppStart';
+
+function getDataStoreFromApi() {
+    const api = getApi();
+    return api
+        .get('dataStore/capture/useNewDashboard');
+}
+
+const getUserDataStoreFromApi = () => {
+    const api = getApi();
+    return api.get('userDataStore/capture/useNewDashboard');
+};
+
+export const fetchDataStoreEpic = (action$: InputObservable) =>
+    action$.pipe(
+        ofType(
+            appStartActionTypes.APP_LOAD_SUCESS,
+        ),
+        mergeMap(async () => {
+            const apiDataStore: UseNewDashboard = await getDataStoreFromApi();
+            // $FlowFixMe
+            return fetchDataStore({ dataStore: apiDataStore });
+        }),
+        catchError(() => EMPTY),
```

Comment:
Does it make sense to log the error here using `log.error `and `errorCreator`?

Reply:
Since this is an "experimental feature", we don't really want to show this as an error in the console. The most common thing is that this dataStorage is not available since not many programs will use the new dashboard with this version. @Bekkalizer feel free to correct me if I'm wrong 👍

Reply:
Yeah the key will usually not be in the dataStore, and we don't need a warning/error in the logs I think...

Reply:
Ok, thanks for the explanation!